icu4j/main/classes/core/src/com/ibm/icu/impl/number/MicroProps.java

```diff
@@ -1,13 +1,25 @@
 // © 2017 and later: Unicode, Inc. and others.
 // License & terms of use: http://www.unicode.org/copyright.html
+
+
 package com.ibm.icu.impl.number;
 
 import com.ibm.icu.number.IntegerWidth;
 import com.ibm.icu.number.NumberFormatter.DecimalSeparatorDisplay;
 import com.ibm.icu.number.NumberFormatter.SignDisplay;
 import com.ibm.icu.number.Precision;
 import com.ibm.icu.text.DecimalFormatSymbols;
+import com.ibm.icu.util.Measure;
+import com.ibm.icu.util.MeasureUnit;
 
+import java.util.ArrayList;
```

Comment:
Unused.

Reply:
Removed in 55c783d3b5, together with an unused iimport com.ibm.icu.util.Measure in FormattedNumber.java